webview/WebView2_spec.md

```diff
@@ -0,0 +1,247 @@
+# Background
+
+The WebView2 is the WinUI 3.0 version of the WebView control. It includes usage of the updated Microsoft Edge browser based on the Chromium web engine.
+<br> 
+<br>
+[Documentation for the WinUI 2.0 version of the WebView control](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.WebView)
```

Comment:
>[Documentation for the WinUI 2.0 version of the WebView control] [](start = 0, length = 64)

The old WebView is not specific to WinUI 2, I think it's better to refer to that as the EdgeHTML based WebView